v0.92-988/non-regression.sh

```diff
@@ -173,15 +198,8 @@ function jerasure_action() {
     non_regression $action "$@" || return 1
     if test "$action" = --check ; then
         path=$(ceph_erasure_code_non_regression --show-path "$@")
```

Comment:
you can get rid of path= because the only reason it exists is to allow different plugin names to verify the same directory. Since you now select the code path with the environment, there is no need for this additional complexity anymore.